shared/src/model/exercise/answer.ts

```diff
@@ -1,12 +1,13 @@
 import Question from './question'
 import { isEmpty } from 'lodash';
 import {
-    BaseModel, modelize, field, computed, getParentOf,
+    BaseModel, modelize, field, computed, getParentOf, ID, NEW_ID,
 } from '../../model';
 
 
 export default
 class ExerciseAnswer extends BaseModel {
+    @field id: ID = NEW_ID
```

Comment:
I'm surprised this wasn't here before. Didn't we need this to save the selected answer too?

Reply:
That was surprising to me as well, it seems it was still getting set on the object though so it worked anyway.